tests/forking_agent/ipc_commands.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright 2025 Inmanta
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Contact: code@inmanta.com
+"""
+
+## This is a separate file to cause the remote side to not load too much code
```

Comment:
This caused a warning by loading test utils in the child process, which loads `pip2pi`  which uses pkg_resource, which causes a warning to travel over the IPC link and break stuff....